client/state/data-layer/wpcom/sites/activity/from-api.js

```diff
@@ -23,7 +23,13 @@ export const DEFAULT_GRIDICON = 'info-outline';
  * @returns {object}             Object with an entry for proccessed item objects and another for oldest item timestamp
  */
 export function transformer( apiResponse ) {
-	return get( apiResponse, [ 'current', 'orderedItems' ], [] ).map( processItem );
+	const response = {
+		totalItems: apiResponse.totalItems,
+		totalPages: apiResponse.totalPages,
+		page: apiResponse.page,
```

Comment:
We add to the object we transform the data received to detect the total pages and the current page of the response to the request.